raiden/tests/utils/smartcontracts.py

```diff
@@ -84,26 +87,61 @@ def deploy_service_registry_and_set_urls(
         service_registry_address=service_registry_address,
         contract_manager=contract_manager,
     )
+    token_address = decode_hex(c1_service_proxy.token_address(block_identifier="latest"))
```

Comment:
@pirapira this line (and maybe others) need adjusting for the tests to pass now that token_address is also bytes

Reply:
Interesting.  `decode_hex` is maybe not type annotated.

Reply:
Not in the eth_utils version we're using. Unfortunately, upgrading to a version with type annotations is not trivial.